src/cephadm/cephadm

```diff
@@ -67,6 +67,13 @@ LOCK_DIR = '/run/cephadm'
 LOGROTATE_DIR = '/etc/logrotate.d'
 SYSCTL_DIR = '/usr/lib/sysctl.d'
 UNIT_DIR = '/etc/systemd/system'
+CEPH_CONF_DIR = 'config'
+CEPH_CONF = 'ceph.conf'
+CEPH_PUBKEY = 'ceph.pub'
+CEPH_KEYRING = 'ceph.client.admin.keyring'
+CEPH_DEFAULT_CONF = f'/etc/ceph/{CEPH_CONF}'
+CEPH_DEFAULT_KEYRING = f'/etc/ceph/{CEPH_KEYRING}'
+SHELL_DEFAULT_PUBKEY = f'/etc/ceph/{CEPH_PUBKEY}'
```

Comment:
why rename the conf and keyring constant from `SHELL` to `CEPH` but then make a new constant for the pub key named `SHELL`?

Reply:
fixed